cupy/testing/_attr.py

```diff
@@ -1,30 +1,12 @@
 import os
 
 
-try:
-    import pytest
-    _error = None
-except ImportError as e:
-    _error = e
-
-
-def is_available():
-    return _error is None
-
+from cupy.testing._pytest_impl import is_available, check_available
```

Comment:
`check_available` is not nice to be imported because of the error message, which starts with `f'{__name__} is not available.'`.

Reply:
Good catch!